### ISSUE TITLE
Loading bar improvements

### ISSUE THREAD

**Issue by samreid** [phetsims/joist#354]
The loading bar should be white and match the width of the PhET logo.  For single screen sims, it may need to start partway completed (10%?)

**Comment by samreid** [comment]
On hold until we are back to master after #348

**Comment by samreid** [comment]
Unassigning until November based on discussion in https://github.com/phetsims/special-ops/issues/40

**Comment by pixelzoom** [comment]
@samreid is this a duplicate of #348?

**Comment by samreid** [comment]
Completed as part of #348, closing.
